x-pack/plugins/cases/public/components/connectors/servicenow/servicenow_sir_case_fields.tsx

```diff
@@ -166,115 +168,132 @@ const ServiceNowSIRFieldsComponent: React.FunctionComponent<
-                    compressed
-                    onChange={(e) => onChangeCb('sourceIp', e.target.checked)}
-                  />
-                </EuiFlexItem>
-              </EuiFlexGroup>
-              <EuiFlexGroup>
-                <EuiFlexItem>
-                  <EuiCheckbox
-                    id="malwareUrlCheckbox"
-                    data-test-subj="malwareUrlCheckbox"
-                    label={i18n.MALWARE_URL}
-                    checked={malwareUrl ?? false}
-                    compressed
-                    onChange={(e) => onChangeCb('malwareUrl', e.target.checked)}
-                  />
-                </EuiFlexItem>
-                <EuiFlexItem>
-                  <EuiCheckbox
-                    id="malwareHashCheckbox"
-                    data-test-subj="malwareHashCheckbox"
-                    label={i18n.MALWARE_HASH}
-                    checked={malwareHash ?? false}
-                    compressed
-                    onChange={(e) => onChangeCb('malwareHash', e.target.checked)}
-                  />
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            </>
-          </EuiFormRow>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiFormRow fullWidth label={i18n.PRIORITY}>
-            <EuiSelect
-              fullWidth
-              data-test-subj="prioritySelect"
-              hasNoInitialSelection
-              isLoading={isLoadingChoices}
-              disabled={isLoadingChoices}
-              options={priorityOptions}
-              value={priority ?? undefined}
-              onChange={(e) => onChangeCb('priority', e.target.value)}
-            />
-          </EuiFormRow>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiFormRow fullWidth label={i18n.CATEGORY}>
-            <EuiSelect
-              fullWidth
-              data-test-subj="categorySelect"
-              options={categoryOptions}
-              value={category ?? undefined}
-              isLoading={isLoadingChoices}
-              disabled={isLoadingChoices}
-              hasNoInitialSelection
-              onChange={(e) => onChange({ ...fields, category: e.target.value, subcategory: null })}
-            />
-          </EuiFormRow>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFormRow fullWidth label={i18n.SUBCATEGORY}>
-            <EuiSelect
-              fullWidth
-              data-test-subj="subcategorySelect"
-              options={subcategoryOptions}
-              // Needs an empty string instead of undefined to select the blank option when changing categories
-              value={subcategory ?? ''}
-              isLoading={isLoadingChoices}
-              disabled={isLoadingChoices}
-              hasNoInitialSelection
-              onChange={(e) => onChangeCb('subcategory', e.target.value)}
+  return (
+    <>
+      {showConnectorWarning && (
```

Comment:
The `DeprecatedCallout` has been added. The other code has been wrapped with `<>`